docs/geedocs/5.2.2/answer/3424202.html

```diff
@@ -10,6 +10,7 @@
 <!-- 3424202.html -->
 <div class="container">
 <div class="sidebar1">
+  <p class="sidebar-toc"><a href="../answer/7160002.html">Open GEE 5.2.2</a></p>
```

Comment:
question: since multiple html files were touched to include 'Open GEE 5.2.2' to the right panel, should we add "copyrights 2018 Open GEE Contributors" to them ?

Reply:
Technically, we probably could, but it's such a trivial change I'm not sure that's worth the trouble

Reply:
Besides, I'm hoping that in the future the TOC isn't going to be hard-coded in the HTML itself :smile: